### ISSUE TITLE
Operating on pairs of inputs

### ISSUE THREAD

**Issue by JoshuaMeyers** [gnina/libmolgrid#75]
Hey Guys, nice library! I've enjoyed using libmolgrid to train over protein-ligand binding affinities but I'd now like to operate over pairs of input PDBs (where each pair is assigned a single label). Do you have any pointers on how one might achieve this with libmolgrid while still making use of the structure cache – I am working with pytorch. Thanks in advance

**Comment by dkoes** [comment]
You would have two labels and two structures in your input types file, like this:
`0 1 in1.pdb in2.pdb`

```
provider = molgrid.ExampleProvider()
provider.populate("in.types")
ex = provider.next()
list(ex.labels) # [0.0, 1.0]
len(ex.coord_sets) # 2
```

**Comment by JoshuaMeyers** [comment]
Thanks! That's much easier than I thought, is there a link to some documentation on the 'types' file?

**Comment by dkoes** [comment]
It's explained in the paper (https://arxiv.org/pdf/1912.04822.pdf), but you're right that the explanation should be in the online documentation.
